harness/determined/tensorboard/fetchers/azure.py

```diff
@@ -0,0 +1,76 @@
+import datetime
+import logging
+import os
+import urllib
+from typing import Any, Dict, Generator, List, Tuple
+
+from .base import Fetcher
+
+logger = logging.getLogger(__name__)
+
+
+class AzureFetcher(Fetcher):
+    def __init__(self, storage_config: Dict[str, Any], storage_paths: List[str], local_dir: str):
+        from azure.storage import blob
+
+        connection_string = storage_config.get("connection_string", None)
+        container = storage_config.get("container", None)
+        account_url = storage_config.get("account_url", None)
+        credential = storage_config.get("credential", None)
+
+        if storage_config.get("connection_string", None):
```

Comment:
ditto.  I'm going to stop marking them, please find all of them.

Reply:
Yup, I suppose this is a bad habit.